### ISSUE TITLE
Read messages from dashboard

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#44]

**Comment by lcabrini** [comment]
This is not written in the form of a user story. Pleaase update.

**Comment by BorderITS** [comment]
So how would you call this if were you?

**Comment by lcabrini** [comment]
All user stories are in the form "As a ROLE a want to ACTION". So, more appropriate would be 

> As a user a want to read incoming messages

The user story should never make assumptions about where this will happen. That is up to the implementation to decide. 

Messages is also a bit obscure here. What type of messages are these? Are they messages from other users? Or are the error messages and confirmation messages. Because if they are confirmation/error/warning messages, they would best go on the page the user is on. If not this weird situation could happen.

1. User tries to log in, get's the password wrong. The "authentication failed" message appears on the dashboard, but since the user is logged in they cannot see it.
2. User enters correct details and gets to dashboard. Now they get a confirmation that they failed to log in a few minutes earlier.

**Comment by BorderITS** [comment]
hmmm you have a point there we'll need to figure it out

**Comment by lcabrini** [comment]
Yes, most likely we will have to find terms to separate these different concepts.

**Comment by BorderITS** [comment]
Ok then we hold on this

**Comment by lcabrini** [comment]
In fact, can we remove this issue from the user stories project. We'll have to revisit this somehow.
